.github/workflows/docker-ci.yml

```diff
@@ -17,3 +17,28 @@ jobs:
       - uses: actions/checkout@v2
       - name: Build FLINT Reporting Client
         run: docker build . --file Dockerfile --tag flint_reporting.client
+
+  build-accountabilities-microservice:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./code/services/accountabilities/
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK
+      - uses: actions/setup-java@v2
```

Comment:
This one is wrong here. Refer here: https://github.com/moja-global/FLINT.Reporting/actions/runs/1818371904